spec.html

```diff
@@ -7751,12 +7751,12 @@ <h1>HostCallJobCallback ( _jobCallback_, _V_, _argumentsList_ )</h1>
       <p>HostCallJobCallback is a host-defined abstract operation that takes arguments _jobCallback_ (a JobCallback Record), _V_ (an ECMAScript language value), and _argumentsList_ (a List of ECMAScript language values).</p>
       <p>The implementation of HostCallJobCallback must conform to the following requirements:</p>
       <ul>
-        <li>It must always perform and return the result of ? Call(_jobCallback_.[[Callback]], _V_, _argumentsList_).</li>
+        <li>It must always perform and return the result of Call(_jobCallback_.[[Callback]], _V_, _argumentsList_).</li>
       </ul>
       <p>The default implementation of HostCallJobCallback performs the following steps when called:</p>
       <emu-alg>
         1. Assert: IsCallable(_jobCallback_.[[Callback]]) is *true*.
-        1. Return ? Call(_jobCallback_.[[Callback]], _V_, _argumentsList_).
+        1. Return Call(_jobCallback_.[[Callback]], _V_, _argumentsList_).
```

Comment:
why is the `?` removed here?

Reply:
What is it signaling to the reader, when it's in tail position? Anyway, I'm fine with adding it back for the default implementation. I was mainly removing it from the requirements, since we're certainly not requiring hosts to _have_ to return early on error before doing its cleanup steps.

Reply:
If the return value of the callback is ignored, then i'd expect:
```suggestion
        1. Perform ? Call(_jobCallback_.[[Callback]], _V_, _argumentsList_).
```

what the ? signals to me is that the operation might return abruptly, which is useful context for me when reading.

Reply:
I don't understand this suggestion. It's incorrect to change the default implementation to not return anything in the normal completion case.

Reply:
Gotcha, then the suggestion is invalid :-)

Thanks for adding it back.